cogboard-webapp/src/components/BoardForm/index.js

```diff
@@ -11,6 +11,18 @@ const BoardForm = ({ renderActions, ...initialFormValues }) => {
   const { values, handleChange } = useFormData(initialFormValues);
   const { autoSwitch } = values;
 
+  const handleInput = (e) => {
```

Comment:
I think it can be a functionality duplication here, check:
https://github.com/Cognifide/cogboard/blob/00be3573ac06cc30c2c460ec155005aa79a4dbc9/cogboard-webapp/src/components/helpers/index.js#L16
Anyway this kind of validation function as a handler is not a good idea in my opinion. Handlers should handle the event, validators should determine if something is valid or not, so validator should be pure function returning true or false. @Cich0sza is trying to introduce validation to our forms, so maybe it would be a better idea to introduce one consistent approach once. We should think about this in terms of tickets in JIRA.